test/trip_updates/trip_updates_pub_sub_test.exs

```diff
@@ -184,5 +184,27 @@ defmodule TripUpdates.TripUpdatesPubSubTest do
 
       assert TripUpdatesPubSub.is_it_fresh?(stale_tripupdate, current_time) == false
     end
+
+    test "you broke it" do
```

Comment:
```suggestion
    test "handles nil arrival times" do
```
suggestion (non-blocking): This made me laugh, but maybe could be more descriptive?

Reply:
Oh noooo I meant to change that >_< sorry